### ISSUE TITLE
Wrong condition on CometChatMessageHeader component

### ISSUE THREAD

**Issue by alvarorodriguezestruch** [cometchat-pro/cometchat-pro-vue-ui-kit#15]
### Describe the problem

The wrong parts is on line 188-198 on the CometChatMessageHeader .vue file

```
if (!isUser) {
  avatar = SvgAvatar.getAvatar(
    this.item.uid,
    this.item.name.charAt(0).toUpperCase()
  );
} else {
  avatar = SvgAvatar.getAvatar(
    this.item.guid,
    this.item.name.charAt(0).toUpperCase()
  );
}
```

### What was the expected behavior?

if `!isUser` the `avatar` variable should be setted with the generation of the avatar using the guid instead of the uid.

### Reproduction

Simple use the ui widget CometChatMessages and see that it does not mount the CometChatMessageHeader because of an exception when `type `prop is group and `item`  prop is a group object.

### Environment

> Please provide the following:

- "@cometchat-pro/chat": "^2.4.0"
- cometchat-pro-vue-ui-kit: 2.4.0-1

**Comment by paarthsabharwal** [comment]
Hi @alvarorodriguezestruch , 
Thank you for sharing.
I have flagged this for our team and we are looking into the issue.
We have created an internal issue to track this and will get back to you with progress soon.
Best, 
Paarth

**Comment by alvarorodriguezestruch** [comment]
Hi @paarthsabharwal 

As you let the source code of the ui widgets to be download and included in the project there is no porblem for me to change it on my project, its just for let you know about it :)

Thanks for quick response
Best regards!

**Comment by paarthsabharwal** [comment]
Hi @alvarorodriguezestruch ,
Thank you so much for your feedback.
I am glad you could solve this out at your end. 
Feel free to reach out to us if you need any help- just email us at help@cometchat.com for anything CometChat.

Best, 
Paarth

**Comment by prakash-nakrani** [comment]
This has been fixed as part of v2.4.1-beta1-1. Please update to the latest version.

Best Regards,
Prakash
